cdap-explore/src/main/java/co/cask/cdap/explore/service/hive/BaseHiveExploreService.java

```diff
@@ -789,6 +793,51 @@ public QueryHandle deleteNamespace(Id.Namespace namespace) throws ExploreExcepti
     }
   }
 
+  public QueryHandle execute(Id.Namespace namespace, String[] statements) throws ExploreException, SQLException {
+    Preconditions.checkArgument(statements.length > 0, "There must be at least one statement");
```

Comment:
It cannot happen, but it is better to safeguard here, than to throw an NPE later.